lib/pure/os.nim

```diff
@@ -1716,15 +1716,54 @@ proc createSymlink*(src, dest: string) {.noWeirdTarget.} =
     if symlink(src, dest) != 0:
       raiseOSError(osLastError(), $(src, dest))
 
+when defined(windows):
+  proc getFinalPathNameByHandleW(
+    hFile: Handle,
+    lpszFilePath: WideCStringObj,
+    cchFilePath: DWORD,
+    dwFlags: DWORD
+  ): DWORD {.importc: "GetFinalPathNameByHandleW",
+    stdcall, dynlib: "Kernel32.dll".}
+
 proc expandSymlink*(symlinkPath: string): string {.noWeirdTarget.} =
   ## Returns a string representing the path to which the symbolic link points.
-  ##
-  ## On Windows this is a noop, `symlinkPath` is simply returned.
+  ## 
```

Comment:
should specify that only 1 level of symlinks are resolved (is that the case on windows too?)